resources/fixtures/config/barcode.yml

```diff
@@ -1,3 +1,9 @@
 file-type: 'png'
 barcode-type: 'code39'
+
+# determines the class that will be used to automatically assign barcodes to products, will typically be the same as
+# the barcode-type
+barcode-generator: 'code39'
```

Comment:
I feel like there is scope for someone to change one of them and overlook the other and get really confused. Esp. devs just prototyping with and testing mothership's features. If it's "typically going to be the same", then why should they need to change two things?

It doesn't matter enough for it to fail QA, so can you just quickly extend the comment to say "If left blank, will default to barcode-type" or similar?

Do we have documentation around barcodes and barcode generation?